plugins/GSdx/GSSetting.cpp

```diff
@@ -33,11 +33,16 @@ const char* dialog_message(int ID, bool* updateText) {
 		case IDC_FILTER:
 			return "Control the texture filtering of the emulation.\n\n"
 				"Nearest:\nAlways disable interpolation, rendering will be blocky.\n\n"
+				"Bilinear Forced/Upscale:\nAlways enable interpolation except for sprite. Rendering is smoother but it could generate few glitches.\n\n"
```

Comment:
`Forced/Upscale` sounds confusing to me. I'd prefer something like `Forced (excluding sprite)` , which is closer to what it's called in the code.

`Rendering is smoother but it could generate few glitches.`
There should be an`a` between generate and few:
`Rendering is smoother but it could generate a few glitches.`